### PR TITLE
update snakemake episode

### PR DESCRIPTION
**Pull request by wikfeldt**
formatting changes and introduction of new optional snakemake+conda exercise

### CODE REVIEW

File: _episodes/06-workflow-management.md [hunk 1]
```diff
@@ -323,7 +323,7 @@ rule NAME:
 ```
 - It is possible to address and offload to non-CPU resources:
 ```
-$ snakemake clean
```
Comment (rkdarst) [inline _episodes/06-workflow-management.md:326]: For my own interest, why is this?

**Comment by rkdarst** [comment]
Looks good, makes me want to use snakemake myself even more now...

File: _episodes/06-workflow-management.md (hunk 1 above)
Comment (wikfeldt) [reply _episodes/06-workflow-management.md:326]: why was "snakemake clean" replaced by "snakemake --delete-all-output" ? It's considered better practice I believe. One can always forget to add things to the clean rule, while --delete-all-output will detect and delete all targets

Comment (rkdarst) [reply _episodes/06-workflow-management.md:326]: Ah, of course... `clean` is a rule, but `--delete-all-output` is a snakemake builtin that is auto-detected.

**Pull request merged**